src/solvers/smt2_incremental/smt_bit_vector_theory.h

```diff
@@ -20,6 +20,56 @@ class smt_bit_vector_theoryt
   static const smt_function_application_termt::factoryt<the_name##t> the_name;
 #include "smt_bit_vector_theory.def"
 #undef SMT_BITVECTOR_THEORY_PREDICATE
+
+  struct additiont final
+  {
+    static const char *identifier();
+    static smt_sortt return_sort(const smt_termt &lhs, const smt_termt &rhs);
+    static void validate(const smt_termt &lhs, const smt_termt &rhs);
+  };
+  static const smt_function_application_termt::factoryt<additiont> addition;
+
+  struct subtractiont final
+  {
+    static const char *identifier();
+    static smt_sortt return_sort(const smt_termt &lhs, const smt_termt &rhs);
+    static void validate(const smt_termt &lhs, const smt_termt &rhs);
+  };
+  static const smt_function_application_termt::factoryt<subtractiont>
+    subtraction;
+
+  struct multiplicationt final
+  {
+    static const char *identifier();
+    static smt_sortt return_sort(const smt_termt &lhs, const smt_termt &rhs);
+    static void validate(const smt_termt &lhs, const smt_termt &rhs);
+  };
+  static const smt_function_application_termt::factoryt<multiplicationt>
+    multiplication;
+
+  struct divisiont final
```

Comment:
:no_entry_sign: We actually need 2 variations of this operation because there is the unsigned divide `bvudiv` and the signed divide `bvsdiv`.